### DISCUSSION TITLE
请求帮助分析 Sodium 导致的崩溃

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#3387] [提问] [answered]
[错误报告-2024-2-20_18.33.14.zip](https://github.com/Hex-Dragon/PCL2/files/14343690/-2024-2-20_18.33.14.zip)
这几天服务器群里有好几个人来问，网友们帮帮忙出个解决方案

更多信息：来反馈的人用的电脑都是 N 卡

**Comment by KLONOA9X6** [comment, marked as answer]
> [错误报告-2024-2-20_18.33.14.zip](https://github.com/Hex-Dragon/PCL2/files/14343690/-2024-2-20_18.33.14.zip) 这几天服务器群里有好几个人来问，网友们帮帮忙出个解决方案
> 
> 更多信息：来反馈的人用的电脑都是 N 卡

关闭微星小飞机
Sodium不兼容RTSS
如果非要用的话在JVM参数里面加
`-Dsodium.checks.win32.rtss=false`
即可在开启RTSS的情况下开启游戏

**Comment by allMagicNB** [comment]
```
[18:33:08] [Render thread/INFO]: 服务端按键监听 Author: QQ448XXX139
```
你这个客服应该考虑下别人的隐私啊（

**Reply by MoYuan-CN** [reply]
.. 那Mod作者写的 关我啥事

**Reply by allMagicNB** [reply]
那你泄露了作者的隐私（
